tasks/cluster-power.yml

```diff
@@ -0,0 +1,153 @@
+---
```

Comment:
The helper node is specific for the helper node setup and not for OCP install. Atleast that's the current model.
These install playbooks are not needed. I'll let @christianh814 comment on it

Reply:
Correct, the current model is to set up a node that will satisfy all the prerequisites for installing OpenShift. It's not meant to do the installation for you.

Automation can be done around the helpernode (Like the [vsphere automation playbook](https://github.com/RedHatOfficial/ocp4-vsphere-upi-automation)), but not with the helpernode itself.

Reply:
Removed the cluster scripts, done.